beetsplug/subsonicupdate.py

```diff
@@ -90,18 +97,37 @@ def __format_url():
                 context_path = ''
             url = "http://{}:{}{}".format(host, port, context_path)
 
-        return url + '/rest/startScan.view'
+        return url + '/rest/{}'.format(endpoint)
+
+    def get_version(self):
+        url = self.__format_url("ping.view")
```

Comment:
```suggestion
        url = self.__format_url("ping")
```

Docs have `.view`, but I don't believe we need `.view`. This is also for the `starScan.view`.

Reply:
like I say before I have put that way because .view work with classic subsonic but not with supysonic
 I will contact maintainer to know if we can add endpoint without .view. so for me it's better to let .view for now no?

Reply:
Sorry, I didn't see your other comment.

No that's fine. Since `.view` is compatible with both versions, I think this is fine.

Thank you very much ☺️